packages/calcite-components/src/components/modal/modal.e2e.ts

```diff
@@ -13,6 +13,14 @@ describe("calcite-modal properties", () => {
     hidden("calcite-modal");
   });
 
+  describe("openClose", () => {
+    openClose("<calcite-modal></calcite-modal>", "open", false);
```

Comment:
If you update `openClose` to use `simplePageSetup` (similar to other test helpers), it will allow devs to set up a test by passing the tag directly in this case. 

```ts
openClose("calcite-modal", "open", false);
```